glossary.md

```diff
@@ -39,3 +39,21 @@ USA -- typeOf ------------> Country
 USA -- name --------------> United States of America
 USA -- containedInPlace --> northamerica
 ```
+
+- **[Measurement method](https://datacommons.org/browser/measurementMethod)**
+
+The technique used for measuring a statistical variable. Describes how a measurement is made, whether by count or estimate or some other approach. May name the group making the measurement to indicate a certain organizational method of measurement is used. Examples include `BosniaCensus` and [`WorldHealthOrganizationEstimates`](https://datacommons.org/browser/WorldHealthOrganizationEstimates).
```

Comment:
Can we include a census example here too? https://datacommons.org/browser/dc/gg17432

And let's remove the BosniaCensus example (the mmethod there is really EurostatRegionalPopulationData)
https://browser.datacommons.org/browser/dc/o/8tv724ddsybr8

Reply:
I replaced the Bosnia Census with the American Community Survey, as linked. I also added a note about nodes possibly containing multiple measurement methods.